source/PluginDev/Assets/GooglePlayGames/Editor/template-AndroidManifest.txt

```diff
@@ -8,6 +8,9 @@
     
     <uses-sdk android:minSdkVersion="8" android:targetSdkVersion="16" /> 
  
+    <uses-permission android:name="android.permission.GET_ACCOUNTS"/>
```

Comment:
sure.  I think the easiest way is to have a static string with the permission elements and then a placeholder in the template which is either replaced with the static string or an empty string.